src/05-led-roulette/README.md

```diff
@@ -10,15 +11,10 @@ I'm going to give you a high level API to implement this app but don't worry we'
 stuff later on. The main goal of this chapter is to get familiar with the *flashing* and debugging
 process.
 
-Throughout this text we'll be using the starter code that's in the [discovery] repository. Make sure
```

Comment:
Checking out the repository will still be necessary / make things easier, right?

Reply:
Yes, I already mentioned that in an earlier chapter exactly because of this^^

https://github.com/rust-embedded/discovery/blob/rewrite/src/03-setup/README.md#this-repository